src/IconButton/index.js

```diff
@@ -12,8 +12,6 @@ export const VALID_ICON_NAMES = iconSelection.icons.map(
  *
  * Icon Button renders a button with a central Narmi icon
  *
- * This component supports rest props; any additional props on icon button will be
- * passed through to button element.
```

Comment:
removing - not actually true: see https://github.com/narmi/design_system/pull/940#discussion_r1231586350